cmd/remoteproc-runtime/root.go

```diff
@@ -35,6 +35,8 @@ func Execute() error {
 
 func init() {
 	rootCmd.PersistentFlags().StringVar(&logLevel, "log-level", "info", "Set the logging level (trace, debug, info, warn, error, fatal, panic)")
+
+	ignoreUnsupportedPodmanFlag(rootCmd)
```

Comment:
Wondering if it's worth adding some logging here. In other words, if log level is debug, then print that we've received this argument but will ignore it. 🤷